Enhance README and add specific logo (#108)
comit-network/comit-js-sdk: README.md

> <a href="https://comit.network">

Author: Note: this link is to be updated to point to the SDK documentation section once it exists.

<img src="logo.svg" height="120px">
</a>

---

[COMIT](https://comit.network) is an open protocol facilitating cross-blockchain applications.

> With [COMIT](https://comit.network) you can for examples exchange Bitcoin for any Erc20 token directly with another person.

Reviewer: English grammar + small improvement proposal (feel free to edit): 
```suggestion
For example, with [COMIT](https://comit.network) you can exchange Bitcoin for Ether or any ERC20 token directly with another person.
```

Reviewer 2: FWIW, I like @bonomat's edit.

> This repository contains the Javascript software development kit (SDK) that wraps the communication with the [comit-network daemon](https://github.com/comit-network/comit-rs) for Javascript developers.

Reviewer: Duplicated information - proposed change: 
```suggestion
This repository contains the Javascript software development kit (SDK) that wraps the communication with the [comit-network daemon](https://github.com/comit-network/comit-rs).
```

If you wish to do an atomic swap on your machine or to integrate COMIT into an application (e.g. a DEX) please take a look at the [Getting Started section](https://comit.network/docs/getting-started/create-comit-app/) of the COMIT documentation.

> If you have any question please [reach out to the team in our Gitter chat](https://gitter.im/comit-network/community)!

Author: ```suggestion
If you have any questions, feel free to [reach out to the team in our Gitter chat](https://gitter.im/comit-network/community)!
```

[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
[![Gitter chat](https://badges.gitter.im/gitterHQ/gitter.png)](https://gitter.im/comit-network/community)

# COMIT Javascript SDK

The COMIT Javascript SDK enables the integration of the [comit-network daemon](https://github.com/comit-network/comit-rs) into Javascript applications.
You'll find the [latest comit-sdk version on npmjs](https://www.npmjs.com/package/comit-sdk).

> Below you find a short introduction into the SDK. For more detailed information please refer to the [API-documentation](TODO-add-link) in our documentation.

Reviewer: Don't forget to track this TODO somewhere else. It's easy to lose track of this.

Reviewer 3: Might be better to remove it until we have the final links.

> Below you find a short introduction into the SDK. For more detailed information please refer to the [API-documentation](TODO-add-link) in our documentation.

Author: ```suggestion
Below you find a short introduction into the SDK. For more detailed information please refer to the API-documentation in our documentation.
```

## SDK Overview

The SDK enables you to create Javascript applications on top of COMIT.
This can for example be a trustless decentralized exchange (DEX) or a multi-currency wallet. 

The SDK offers functionality to negotiate a trade and to execute the trade after the negotiation is finished.

### Trade Negotiation vs. Execution

The SDK integrates a simple negotiation protocol that serves as a starting point for makers and takers to create and take orders.

> It is recommended to take a look at the [`TakerNegotiator`](https://github.com/comit-network/comit-js-sdk/blob/master/src/negotiation/taker_negotiator.ts) and [`MakerNegotiator`](https://github.com/comit-network/comit-js-sdk/blob/master/src/negotiation/maker_negotiator.ts) classes to get started.

Reviewer: I would remove this sentence (or describe how to use it) as I would not know what to look at in these two files _to get started_.
```suggestion
```

Author: please see my comments below :)

> It is recommended to take a look at the [`TakerNegotiator`](https://github.com/comit-network/comit-js-sdk/blob/master/src/negotiation/taker_negotiator.ts) and [`MakerNegotiator`](https://github.com/comit-network/comit-js-sdk/blob/master/src/negotiation/maker_negotiator.ts) classes to get started.

Author: ```suggestion
The [`MakerNegotiator`](https://github.com/comit-network/comit-js-sdk/blob/master/src/negotiation/maker_negotiator.ts) class gives the maker the possibility to create and publish simple `JSON` orders.
The [`TakerNegotiator`](https://github.com/comit-network/comit-js-sdk/blob/master/src/negotiation/taker_negotiator.ts) class enables the taker to connect to the maker and take a published order. 
```

Author: @bonomat how about this?

Author: Note: After discussion with @D4nte: 

The README is planned to serve as starting point for the documentation of the SDK, hence I would keep the information about `MakerNegotiator`, `TakerNegotiator` and `ComitClient` in there for now. If we decide to move the starting point of the SDK documentation to be separate from the README we can shorten the README - for now I would keep this information in here as an overview of the SDK (the information is in one place only at the moment, in the README).

Author: since everyone else approved I'll just merge this PR for now. Follow-up PR if changes are needed.

* Bitcoin wallet based on [bcoin](https://github.com/bcoin-org/bcoin)
* Ethereum wallet based on [ethers.js](https://github.com/ethers-io/ethers.js/)

These wallets are used to actually send the transactions necessary for the atomic swap to the respective ledger.
COMIT is designed in a non-custodial way, thus `cnd` never holds your private key at any time.
The wallets integrated in the SDK show how the transaction information received from `cnd` can be sent to the respective ledger.

> Note that you can, of course, use different wallets then the above default wallets in your application.

Reviewer 2: ```suggestion
Note that you can, of course, use different wallets than the above default wallets in your application.
```
